docs/relying-parties/reference/metrics-for-relying-parties.md

```diff
@@ -4,7 +4,29 @@ title: Metrics for Relying Parties
 sidebar_label: Metrics
 ---
 
-Last updated: `January 26th, 2022`
+Last updated: `March 20th, 2023`
+
+## Subscriptions Services Data Sources
+Subscription Platform provides product metrics by obtaining subscriptions data from the following sources: Stripe, mobile app store providers, and FxA logs.
+
+### Stripe
+The bulk of Subscriptions Services data comes from Stripe and is synced to BigQuery using Fivetran and its [Stripe connector](https://fivetran.com/docs/applications/stripe). This data reflects the current subscription state of all active subscriptions paid for via Stripe or PayPal. It also includes information regarding subscriptions and customers, as well as products, plans, invoices, cards, charges, discounts, coupons, and promotion codes.
+
+Some of the customer information includes the customer's postal code, state, and country for a subscription, which helps resolve issues for customers across a particular dimension (e.g., country, payment provider, etc.). However, as it intentionally excludes any personal identifiable information of customers (e.g., name, street address, phone number, email address), it can be challenging to use to help on an individual customer level. Other uses for this data is for retrieving all-time counts (e.g., total number of active subscriptions).
+
+Fivetran preserves historical data within `subscription_history` by comparing the current state of the subscription with the most recent record in the history table and adding a new record if they vary.
+
+### Mobile app store providers
+Subscriptions Services data is also obtained from two mobile app store providers: Apple and Google. This includes date of billing and expired dates, subscription change event, etc. Apple provides limited data - it does not provide the country, price, or currency of subscriptions.
+
+Subscription Platform handles getting Apple subscriptions from the Apple API and stores the latest state of active subscriptions in Firestore. Prior to December 2022, VPN Apple subscriptions data was managed by VPN's Guardian service and stored in the Guardian database. As Firestore doesn't contain the full historical state of Apple subscriptions, Guardian data is still used for historical purposes.
+
+Similar to Apple, Subscription Platform also handles the current and historical state of Google subscriptions and it is synced to Firestore as well.
```

Comment:
Getting a bit technical here, but only current state for IAP subscriptions (Apple and Google) are stored in Firestore. We have a BigQuery sync that syncs these two collections and a separate table for each that syncs the changelogs, which is where we can get historical state from.

Reply:
Getting more technical:  the changelog for each collection is what gets synced to a BigQuery table, and there's a BigQuery view for each that represents the current state of the collection calculated based on the latest changelog for each document.

Since the historical state/changes are only recorded by the BigQuery sync itself, we only have records of the changes that occurred after the BigQuery syncs were set up, so it might be worth noting when those syncs were set up:
- Google IAP sync started 2021-10-18.
- Apple IAP sync started 2022-12-15.